### DISCUSSION TITLE
Sparkle will show update notification though settings shouldn't permit

### DISCUSSION THREAD

**Discussion by quickiwiki** [sparkle-project/Sparkle#2500] [Q&A]
## Summary
Update messages are supressed in the settings and application with the Sparkle has downloaded the update and is waiting to be restarted, then another instance is launched and the Sparkle will show the notification that "A new version of %APP_NAME% is ready to install" though it shouldn't according to the settings

## Possible Fix

[not obligatory, but suggest fixes or reasons for the bug]

## Version

2.4.1

**Comment by zorgiepoo** [comment]
This report is confusing to understand but you are probably running into expected behavior.

Assuming you turned on automatic downloading/installing of updates which is unclear here, read the [documentation for SUAutomaticallyUpdate](https://sparkle-project.org/documentation/customization/) which notes that an update alert may be presented 1 week after the update has been downloaded if the user has not quit the app yet. An update alert may also be presented if the update requires user authorization (their password) to install/replace the older version. An update alert may also be presented earlier if an update is explicitly marked as critical or informational only or a major upgrade.

If you are running into something else please clarify with a list of detailed steps or list of actions that occur.
